### ISSUE TITLE
2x2 Trees Only Bonemeal At Superflat Height, Don't Respond To Tickspeed / Certain 1x Saplings Have Same Problems

### ISSUE THREAD

**Issue by JustAMutant** [Potion-Studios/Oh-The-Biomes-Weve-Gone#180]
### Additional Bug Found 12/11/24
Loader: Forge
Most Similar Bug Report: #154 
*( we likely have the same exact root cause, feel free to merge this report as needed )*

### Initial First Test

**Tectonic** Test
Mod 1.4.4 / MC 1.20.1 

### BWG Superflat Test(s)
MC 1.21.1 *( why did I test this using forge & not neo )*
Mod 2.2.5 / 2.2.4 / 2.2.3 / 2.2.2 

MC 1.20.1 *( forge )*
1.4.4 / 1.4.3 / 1.4.2 / 1.2.3 BWG Official Release Version / 1.1.2 Beta / 1.0.1 Beta *( +1 version up from snapshot )*

### Other Bug
MC 1.20.1 / BWG 1.4.4 / TYG 1.3.1
1x Aspen was found to respond to tickspeed increases & bonemeal just fine, regardless of world height, even at 313 blocks.
**Redwood, Ebony, Baobab & Blue Enchanted** were the 4 staple test subjects for the entirety of all tests I ran + unresponsive 1x's past superflat worldheight.

1x Saplings of the standard tested species, **didn't respond to tickspeed increases past superflat height**, nor did they take bonemeal, indicating they _MAY_ possibly NOT grow AT ALL, even when planted as 1x past superflat.

### TYG Superflat Test(s)
1.30 TYG + 1.4.2 BWG / 1.2.3 BWG 

1.20 **Beta** TYG + 1.0.4 **Beta** BWG
Blue enchanted consumed bonemeal even in creative, crashed after I kept bonemealing it whilst other tested trees did not, crash report;
[crash-2024-12-11_17.59.16-server.txt](https://github.com/user-attachments/files/18104793/crash-2024-12-11_17.59.16-server.txt)

Superflat tests were done with nothing but required mods to run BWG, config folder was deleted each time.

### Tectonic Yapping / Original First Tests
Tectonic is a popular mod to pair with BWG, an option it has is a build height increase which is compatible with BWG's world gen, but 2x2 saplings **don't respond to bonemeal** past at a mere **17 blocks in elevation.**

This lead to going on a downward spiral after looking at bugreport #154, as upon further testing;
**It's NOT just Tectonic, ALL 2x2 saplings hate world height, INCLUDING small ones like Blue Enchanted.**

For additional context on how bad 17 blocks not working is, Howling Peaks from BWG with additional height enabled, can range from 300-500 blocks tall & 17 is **UNDER AVERAGE SEA LEVEL** with this setting enabled.

Various Heights Tested; *( both using Tectonic & 
Superflat height at *minus* -60 works fine with 2x2 trees
17, 30 & 50 Blocks do not respond
80, 100, 150 & 214 Also do not work

Tested with just BWG's latest 1.21.1 version + a superflat world, with blue enchanted not generating hammering in this bugs REAL root cause, as it's a very small 2x2 tree in comparison to #154  + other trees tested during my first breakdown with Tectonic.
<img width="1280" alt="2024-12-10 18_06_25-" src="https://github.com/user-attachments/assets/ee285436-777a-464a-8936-769fcae3c5a4">
Where to find increased height in configs & other evidence; *( probably easier with mod menu )*
<img width="760" alt="2024-12-10 17_38_35-tectonic json - Visual Studio Code" src="https://github.com/user-attachments/assets/2deec2af-62a8-4c79-9c65-9d02a1baaf3d">

**Comment by JustAMutant** [comment]
Actually finished with any changes done to this bug report, apologies if it's a bit hard to read in some areas as a result of all the version tests.

**Comment by CorgiTaco** [comment]
@JustAMutant This wont work bc the tree is not supported. You'll need a bigger dirt area for the tree to support itself as the trunk builders/depth dont allow it to "float". I know this is a bit confusing but we decided that the traditional 2x2 and 1x1 sapling shapes were better than having users plant saplings in the shape of the trunk. There are no mod incompatibilities either

Video of that here: https://i.imgur.com/IJsKO1O.mp4

Screenshot of Y height: 

![Image](https://github.com/user-attachments/assets/4e22d0a8-20c0-48db-8190-9b9e7cc6eab3)

**Comment by CorgiTaco** [comment]
TLDR dont use a 2x2 platform for growing trees with larger than 2x2 trunks
